bdsf/plotresults.py

```diff
@@ -369,24 +363,25 @@ def plotresults(img, ch0_image=True, rms_image=True, mean_image=True,
             plot_sed(sed_src, ax)
 
         elif image == 'wavelets':
-            if i == index_first_waveplot:
-                for j in range(len(j_with_gaus)):
-                    cmd = 'ax' + str(j+i+1) + ' = pl.subplot(' + str(numx) + \
-                        ', ' + str(numy) + ', ' + str(j+i+1) + ', sharex=ax1, '+\
-                        'sharey=ax1)'
-                    exec(cmd)
-                    pl.title('Pyramidal Sources for\nWavelet Scale J = ' +
-                             str(j_with_gaus[j]))
-            for pyr in img.pyrsrcs:
-                for iisl, isl in enumerate(pyr.islands):
-                    jj = pyr.jlevels[iisl]
-                    jindx = j_with_gaus.index(jj)
-                    col = colours[pyr.pyr_id % 6]
-                    ind = N.where(~isl.mask_active)
-                    cmd = "ax" + str(jindx + index_first_waveplot + 1) + \
-                        ".plot(ind[0]+isl.origin[0], "\
-                        "ind[1]+isl.origin[1], '.', color=col)"
-                    exec(cmd)
+            print('Pyramidal source plots not yet supported.')
+            # if i == index_first_waveplot:
+            #     for j in range(len(j_with_gaus)):
+            #         cmd = 'ax' + str(j+i+1) + ' = pl.subplot(' + str(numx) + \
+            #             ', ' + str(numy) + ', ' + str(j+i+1) + ', sharex=ax1, '+\
+            #             'sharey=ax1)'
+            #         exec(cmd)
+            #         pl.title('Pyramidal Sources for\nWavelet Scale J = ' +
+            #                  str(j_with_gaus[j]))
+            # for pyr in img.pyrsrcs:
+            #     for iisl, isl in enumerate(pyr.islands):
+            #         jj = pyr.jlevels[iisl]
+            #         jindx = j_with_gaus.index(jj)
+            #         col = colours[pyr.pyr_id % 6]
+            #         ind = N.where(~isl.mask_active)
+            #         cmd = "ax" + str(jindx + index_first_waveplot + 1) + \
+            #             ".plot(ind[0]+isl.origin[0], "\
+            #             "ind[1]+isl.origin[1], '.', color=col)"
+            #         exec(cmd)
```

Comment:
No, just to make it easier to add the functionality in case we decide to. I must say though that that is probably quite unlikely, as I've never heard anyone express an interest in the pyramidal source stuff (and I think it would be a fair bit of work to get it all working).